client/src/data_layer_configs/lead_service_lines_by_water_systems_config.ts

```diff
@@ -114,22 +115,23 @@ const popupInfo: PopupInfo = {
       dataType: FeaturePropertyDataType.String,
       optional: true,
     },
-    {
-      label: 'Number of lead connections',
-      name: 'lead_connections_count',
-      dataType: FeaturePropertyDataType.Number,
-    },
-    {
-      label: 'Number of service lines',
-      name: 'service_connections_count',
-      dataType: FeaturePropertyDataType.Number,
-    },
     {
       label: 'Population served',
       name: 'population_served',
       dataType: FeaturePropertyDataType.Number,
     },
   ],
+  computedProperties: [
+    {
+      label: 'Percent lead lines',
```

Comment:
Let's make this `Estimated percent lead lines` per [this](https://tables.area120.google.com/table/bX_kMNBW8LA35n6l5Hz_co/row/aVsp8DFDsMieWlDzLIJk8H).

(though I still don't love this phrasing. But something explicit like  `Estimated fraction of water service lines that contain lead` feels too long. I'll see if someone from branding can look through and possibly edit the text throughout the app)

Reply:
Done!